src/main/java/htsjdk/variant/vcf/VCFHeaderLineCount.java

```diff
@@ -25,9 +25,78 @@
 
 package htsjdk.variant.vcf;
 
+import htsjdk.utils.Utils;
+
 /**
  * the count encodings we use for fields in VCF header lines
  */
+//TODO: this should really be called VCHFHeaderCountType
```

Comment:
Because this PR is introducing breaking changes anyway, I vote for change the name here too.